packages/nimble-components/src/toggle-tip/specs/toggle-tip-ixd.md

```diff
@@ -0,0 +1,147 @@
+# Nimble Toggletip (IxD)
+
+## Overview
+
+A toggletip is a web component that provides a tooltip-like functionality with the ability to toggle its visibility. It is typically used to display additional information or context for a specific element.
+
+The toggletip consists of a trigger button and a content panel that is hidden by default. When the trigger button is clicked, the content panel is revealed, providing the user with additional information or options. Clicking outside the toggletip or pressing escape will hide the content panel again.
+
+Toggletips are commonly used in user interfaces to provide contextual help, explanations, or additional actions without cluttering the main interface.
+
+![Toggletip example](./spec-images/example-toggle-tip.png)
+
+### Background
+
+-   [ViD spec](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=5308-156796&mode=dev)
+-   [Component Issue](https://github.com/ni/nimble/issues/1876)
+
+## Usage
+
+**When to use:**
+
+-   In a _form_ or _editor_ view to display additional information or context for a specific element.
+-   When the additional information or context provided by the toggletip is not valuable for most users of the control.
+
+**When not to use:**
+
+-   When the additional information or context is valuable for most users of the control. This context should be visible by default.
+
+## Anatomy
+
+![Toggletip anatomy](./spec-images/anatomy.png)
+
+| Element        | Description                                                                                        |
+| -------------- | -------------------------------------------------------------------------------------------------- |
+| Trigger button | Info icon button for showing and hiding an associated popover                                      |
+| Popover        | Overlay container for text and interactive components                                              |
+| Content area   | Area consisting of slots for a title, 1-3 lines of text, and a button or link. See the Banner API. |
+
+## Sizing
+
+![Minimal toggletip](./spec-images/minimal-toggle-tip.png)
+
+![Maximal toggletip](./spec-images/maximal-toggle-tip.png)
+
+### Width
+
+The width of the toggletip popover should be determined based on the length of the content it contains. However, if the content is greater than a single line of text, the width should be set to 300px.
+
+### Height
+
+When the popover height exceeds that of the viewport, the popover should not become scrollable to try and fit in the viewport (not pictured). The popover should always be tall enough to show all its content. The surrounding content, for example the page itself, should scroll instead.
+
+Popovers with text should be limited to approximately three lines of text under normal display size at the intended location.
+
+## Alignment
```

Comment:
I'm not sure about the implementation strategy around these requirements..
I wouldn't want to add a bunch of additional complexity on top of what the anchored region supports.

Where does the ordering requirement come from?

Seems like applying these rules as written might be complex (there  is a priority order between fitting in viewport, overlapping "other content", and choosing the highest priority in the cycle of options).

Reply:
The ordering requirement comes from an attempt to "avoid obscuring the content which it annotates". 

Instead of defining the order, could we pass annotated content area along with the viewport to the anchored region and have the anchored region calculate the position? Or default to the top-right position, but still allow the anchored region to move if it doesn't fit in the viewport?

Reply:
Might be a post stand-up topic to discuss how to resolve this and @mollykreis and @atmgrifter00 's input may be helpful to know what anchored-region does today / anchor positioning is capable of tomorrow.
My hunch is preferring a direction and adjusting for the viewport is possible. But not clear what precedence exists / future-dence is coming API-wise.

Reply:
We discussed this briefly on Friday - It's currently unclear how we might add precedence to the anchored-region positioning, but since we can specify the initial toggle-tip direction (MVP), we can negotiate precedence in the HLD. No need to resolve it all in the IxD spec.

Reply:
As long as we capture a note that it is still an open question / needs prototyping that works for me 👍